yt_dlp/extractor/tenplay.py

```
@@ -110,11 +111,21 @@ def _real_extract(self, url):
             f'https://vod.ten.com.au/api/videos/bcquery?command=find_videos_by_id&video_id={data["altId"]}',
             content_id, 'Downloading video JSON')
         # Dash URL 403s, changing the m3u8 format works
-        m3u8_url = self._request_webpage(
-            HEADRequest(update_url_query(video_data['items'][0]['dashManifestUrl'], {
-                'manifest': 'm3u',
-            })),
-            content_id, 'Checking stream URL').url
+        if 'items' in video_data and video_data['items'] and 'dashManifestUrl' in video_data['items'][0]:
+            m3u8_url = self._request_webpage(
+                HEADRequest(update_url_query(video_data['items'][0]['dashManifestUrl'], {
+                    'manifest': 'm3u',
+                })),
+                content_id, 'Checking stream URL').url
```

```suggestion
        if mpd_url := traverse_obj(video_data, ('items', ..., 'dashManifestUrl', {url_or_none}, any)):
            m3u8_url = self._request_webpage(
                HEADRequest(mpd_url), content_id, 'Checking stream URL',
                query={'manifest': 'm3u'}).url
```

---

The suggestion doesn't have the same semantics FWIW, probably moot since the BC URL members are no longer expected. This is betting that if the site were to send a value worthy of `video_data['items']` there might be a valid `dashManifestUrl` in the `[1]` or later items when there wasn't in the `[0]` item; previously and in the proposed change, only `video_data['items'][0]` was checked. Maybe `HLSURL` is worth a bet too?

In any case, the comment above should be updated, s/t like this?
```diff
-        # Dash URL 403s, changing the m3u8 format works
+        # HLSURL no longer available:
+        # * convert any dashManifestUrl to HLS (actual DASH gave 404)
+        # * or use a hard-coded base HLS URL
```
Maybe also make the first case non-fatal and fall back to the hard-coded value?